code/obj/artifacts/artifact_objects/lamp.dm

```diff
@@ -6,6 +6,7 @@
 	var/light_g = 1
 	var/light_b = 1
 	var/datum/light/light
+	var/obj/effect/bonus_light
```

Comment:
Nice, minor thing, should probably be qdel'd in disposing() for GC

Reply:
good catch. I had it qdel_nulling it and then I reorganised it a bit and forgot it D: